subxt/examples/frontier_eth.rs

```diff
@@ -0,0 +1,59 @@
+#![allow(missing_docs)]
+
+use subxt::OnlineClient;
+use subxt_signer::eth::{dev, AccountId20, Signature};
+
+#[subxt::subxt(runtime_metadata_insecure_url = "ws://127.0.0.1:9944")]
+mod eth_runtime {}
```

Comment:
prob worth a comment that in order to run this example, somebody should be running a frontier node locally (and maybe the comment can briefly say how eg `frontier --dev` or whatever it is).